### PR TITLE
Outputs manipulation

### PR DESCRIPTION
**Pull request by gregwis**
Added methods for manipulation of Integra outputs (set_output_on, set_output_off) and monitoring outputs state (_output_changed).

### CODE REVIEW

File: satel_integra/satel_integra.py [hunk 1]
```diff
@@ -211,6 +239,11 @@ def _partition_bytes(self):
         partition = 1 << self._partition_id - 1
         return partition.to_bytes(4, 'little')
 
+    @property
```
Comment (c-soft) [inline satel_integra/satel_integra.py]: Is this method necessary? 
Seems like it's not used anywhere?

**Comment by gregwis** [comment]
I removed unused method.
It was tested with Home Assistant but with my local modification with added support for outputs handled in HA as a switch. I will make a pull request to HA but it will need my modifications in satel_integra (version 0.2.0 of satel_integra).

I haven't tested changes in satel_integra without my modifications in HA. I think it should work but I haven't tested it. I could test it within a few days.

**Comment by gregwis** [comment]
Tests done.
I had to change order of constructor parameters to make it working with current version of Home Assistant (0.61).
Should I make a new Pull Request?

**Comment by gregwis** [comment]
Hi,
could you make a new release 0.2.0 with this pull request merged? I would like to add dependency to Home Assistant with version 0.2.0 of satel_integra.

**Comment by c-soft** [comment]
Hi, I got another merge request with bugfix coming in.
I will merge it and then make a release.
Needs a bit of testing though.

**Comment by pa0los** [comment]
@gregwis Could you provide the modified HA component supporting this feature?

**Pull request merged**